src/ec/mod.rs

```diff
@@ -62,6 +62,7 @@ pub struct KeyPair {
     pub public_key: [u8; PUBLIC_KEY_MAX_LEN],
 }
 
+#[derive(Clone, Copy)]
```

Comment:
I think we don't want to rely on `Copy` semantics always being supported. At least, we'd need a very strong argument to make such a guarantee. In the future I'd like to add transparent HSM support and similar things, and `Copy` wouldn't work for that.